POLMED/src/components/button/index.tsx

```diff
@@ -1,10 +1,10 @@
 type Props = {
     content: string;
-    className: string;
+    className: "btn-primary" | "btn-secondary" | "btn-tertiary" | "btn-warning";
 };
 
 const Button = ({content, className}:Props) => {
-    return <button className={className}>{content}</button>;
+    return <button className={`btn ${className}`}>{content}</button>;
```

Comment:
proponuję zmienić trochę nazewnictwo tego propsa, gdyż słowo className oznacza już samo w sobie nadanie klasy, gdy ktoś będzie z zewnątrz patrzył na komponent rodzica może zastanawiać się czy className to jest juz klasa dla komponentu czy przekazywany props
osobiście proponuję nazwę type | btnType | buttonType